tests/end-to-end/director/application_api_test.go

```diff
@@ -1058,6 +1058,14 @@ func generateSampleApplicationInputWithName(placeholder, name string) graphql.Ap
 	}
 }
 
+func generateSampleRuntimeInputWithName(placeholder, name string) graphql.RuntimeInput {
```

Comment:
Maybe these `generateSampleXXXInputWithName` functions could be moved to fixtures file because that's where they are used.